pkg/kf/testutil/integration.go

```diff
@@ -93,6 +103,123 @@ func RunIntegrationTest(t *testing.T, test func(ctx context.Context, t *testing.
 	test(ctx, t)
 }
 
+// RunKubeAPITest is for executing tests against the Kubernetes API directly.
+func RunKubeAPITest(t *testing.T, test func(ctx context.Context, t *testing.T)) {
+	if !strings.HasPrefix(t.Name(), "TestIntegration_") {
```

Comment:
nit: Now that this is in multiple places, can we break it out to a helper method?